.all-contributorsrc

```diff
@@ -105,6 +105,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "Gab990",
+      "name": "Gabor Sebestyen",
+      "avatar_url": "https://avatars.githubusercontent.com/Gab990",
+      "profile": "https://github.com/Gab990",
+      "contributions": [
+        "code"
+      ]
```

Comment:
this is unnecessary, we add this later using @all-contributors bot after the PR being merge